[ffmpeg] Add 6.0

marcwrobel opened the pull request:
I set this release to LTS, but I do not know what is the rationale behind those LTS versions. I could not find any documentation about it on https://ffmpeg.org/download.html.

Also took the opportunity to normalize the page (#2124).

marcwrobel committed:
[ffmpeg] Normalize page (#2124)

marcwrobel committed:
[ffmpeg] Add 6.0

I set this release to LTS, but I do not know what is the rationale behind those LTS versions. I could not find any documentation about it on https://ffmpeg.org/download.html.

BiNZGi commented on products/ffmpeg.md:
```diff
@@ -3,235 +3,251 @@ title: FFmpeg
 -   releaseCycle: "3.2"
     codename: 'Hypatia'
+    releaseDate: 2016-10-27
     eol: false
```
3.2 is on [old releases](https://ffmpeg.org/olddownload.html) and thus eol on `latestReleaseDate`
```suggestion
    eol: 2022-10-28
```

BiNZGi reviewed (changes requested):
> Moreover the description does not seem in sync (anymore ?)

Good catch, please update our description and remove the `lts` flags.

marcwrobel committed:
[ffmpeg] Set EOL date for 3.2

3.2 is now listed on https://ffmpeg.org/olddownload.html.

Co-authored-by: BiNZGi <BiNZGi@users.noreply.github.com>

marcwrobel committed:
[ffmpeg] Remove LTS flags for all but 5.1 and update description (#2602)

Expectation of yearly and LTS releases comes from (now-dead, archive url) the 5.0 release blog post (http://web.archive.org/web/20220204052947/http://www.jbkempf.com/blog/post/2022/FFmpeg-5.0). It was also covered on Phoronix as "FFmpeg 5.0 Aims To Be An LTS Release" (https://www.phoronix.com/news/FFmpeg-5.0-LTS-Plans).

Looking at release notes, only 5.1 was considered as such : https://git.ffmpeg.org/gitweb/ffmpeg.git/blob/refs/heads/release/5.1:/RELEASE_NOTES.

Co-authored-by: Nemo <commits@captnemo.in>
